apps/backend/src/libs/modules/token/token.ts

```diff
@@ -0,0 +1,11 @@
+import { config } from "../config/config.js";
+import { TokenService } from "./token.service.js";
+
+const token = new TokenService(
+	config.ENV.JWT.SECRET,
```

Comment:
pass as an object

Reply:
Should I create a type for this object? If so, should I put it in token.service.js?